tools/reviewer/web/src/app/dashboard/reviews-panel/file-changes/file-changes.service.ts

```diff
@@ -53,4 +57,28 @@ export class DiffService {
   hoverCheck(i: number, isNewCode: boolean): boolean {
     return this.hoveredLine === i && this.isHoveredNewCode === isNewCode;
   }
+
+  // Get langulage from filename. Example:
+  // filename.js -> javascript
+  getLanguage(filename: string): string {
+    const extensionRegExp: RegExp = /(?:\.([^.]+))?$/;
+    const extension: string = extensionRegExp.exec(filename)[1];
+
+    switch (extension) {
```

Comment:
Can you add a link to the list of possible languages?

Reply:
Done